specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/cloud-events-schema/email_engagement_tracking_report_received.json

```diff
@@ -0,0 +1,15 @@
+{
+  "eventId": "5dc9bc75-1712-4e26-8aa5-769bea7aaf13",
```

Comment:
Verify that the examples match the individual schemas.

Reply:
updated the json sample